gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
All generated files have to be in .gitignore. e.g. wsdl file usually located outside the generated dir, because its an integration contract.